src/cityreader/cityreader.py

```diff
@@ -14,28 +14,48 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
-cities = []
+list_of_cities = []
+
+
+class City():
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __repr__(self):
+        return f"City({repr(self.name)}, {repr(self.lat)}, {repr(self.lon)})"
+
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    # TODO Implement the functionality to read from the 'cities.csv' file
+    # For each city record, create a new City instance and add it to the
+    # `cities` list
+
+    with open('src/cityreader/cities.csv', newline='') as csvfile:
+        citylist = csv.reader(csvfile)
+        next(citylist)
+        for row in citylist:
+            cities.append(City(row[0], float(row[3]), float(row[4])))
```

Comment:
Ahhh there it is. Nice! I was trying to think of a different way to tell you to use floats, but I didn't want to give the answer away! I'm really glad to see you figured it out! Excellent work, my friend. You have passed this Sprint Challenge!
![salute](https://user-images.githubusercontent.com/5281855/66232980-10f21580-e69f-11e9-8df3-bc291a0baaab.gif)